mlir/utils/performance/quickTunerPreproc.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+
+"""
+quickTuner preprocessor script to combine .debug output files from tuningRunner.py or tuna-script.sh
+
+Usage: quickTunerPreprocess.py [-h] --input-dir INPUT_DIR --output OUTPUT [--op {gemm,conv}] [-d] [--file-ext FILE_EXT]
+
+Example Usage:
+
+python3 quickTunerPreprocess.py --input_dir /path/to/debug/files --ouput combined_data 
+
+
+Note:
+If using MITuna edit MITuna/tuna/rocmlir/rocmlir_worker.py, editing:
+
+
+    cmd = env_str + f" python3 ./bin/tuningRunner.py -q {special_args} \
+                     --config='{config_string}' --mlir-build-dir `pwd` \
+                     --output=- --tflops \
+                     --rocmlir_gen_flags='--device={self.gpu_id}' 2>/dev/null"
+
+
+to:
+    
+    import uuid
```

Comment:
Could this instead be a patch to `rocmlir_worker`?

Reply:
Not in its current form;  it cuts off the usual functionality of passing results to stdout and we don't have code to read results from the files and send it on.  I do have an open issue about collecting all the results into the database and this may become part of it.